src/main/java/org/spongepowered/mod/registry/SpongeGameRegistry.java

```diff
@@ -555,7 +555,7 @@ private void setEntityTypes() {
         this.entityTypeMappings.put("PRIMED_TNT", new SpongeEntityType(20, "PrimedTnt", (Class<? extends Entity>)EntityList.stringToClassMapping.get("PrimedTnt")));
         this.entityTypeMappings.put("FALLING_BLOCK", new SpongeEntityType(21, "FallingSand", (Class<? extends Entity>)EntityList.stringToClassMapping.get("FallingSand")));
         this.entityTypeMappings.put("FIREWORK", new SpongeEntityType(22, "FireworksRocketEntity", (Class<? extends Entity>)EntityList.stringToClassMapping.get("FireworksRocketEntity")));
-        this.entityTypeMappings.put("ARMORSTAND", new SpongeEntityType(30, "ArmorStand", (Class<? extends Entity>)EntityList.stringToClassMapping.get("ArmorStand")));;
+        this.entityTypeMappings.put("ARMOR_STAND", new SpongeEntityType(30, "ArmorStand", (Class<? extends Entity>)EntityList.stringToClassMapping.get("ArmorStand")));;
```

Comment:
Was already like that. I'll leave it because there's not much point making one minor fix to the file that has lots of minor issues. I will fix this in https://github.com/SpongePowered/Sponge/pull/112 if I push there again